Command/IndexCreateCommand.php

```diff
@@ -38,7 +38,13 @@ protected function configure()
                 InputOption::VALUE_NONE,
                 'If the time suffix is used, its nice to create an alias to the configured index name.'
             )
-            ->addOption('no-mapping', null, InputOption::VALUE_NONE, 'Do not include mapping');
+            ->addOption('no-mapping', 'nm', InputOption::VALUE_NONE, 'Do not include mapping')
```

Comment:
Options shortcut should have not more than one letter. In this particular case let's leave this option without shortcut.